### ISSUE TITLE
0.21-alpha4 crashes in Google Cloud Kubernetes during import

### ISSUE THREAD

**Issue by Tarpsvo** [meilisearch/MeiliSearch#1402]
**Describe the bug**
I have 14 indexes with entry count ranging from ~100 to ~30 000 items. The issue is that I can't import data using Laravel Scout into the largest indexes (with ~30k items). The Meilisearch crashes only a few requests into the process. The first 10 items are imported and then it crashes without any errors.

I had similar issues with version 0.20. In 0.20, it crashed even when creating indexes.

The data is not nothing special, an example of an entry is as follows:
![image](https://user-images.githubusercontent.com/2018660/121507711-9047f880-c9ed-11eb-8a01-008eaca319dc.png)

Is there an issue with indexes containing different data? The indexes are basically:
5 indexes for an e-commerce project's categories (5 different countries, each country has its own CMS instance)
5 indexes for an e-commerce project's products (5 different countries, each country has its own CMS instance)
2 indexes for a different e-commerce project's categories (development and staging)
2 indexes for a different e-commerce project's products (development and staging)

Basically, the only thing the 14 indexes have in common is the primary key which is ID, otherwise the data can be very different. Could the issue be related to that?

**To Reproduce**
1. Start up Meilisearch 0.21 in Google Cloud Kubernetes Engine
2. Create 10+ different smaller indexes with small number of entries
3. Create and try to fill an index with a lot of entries

**Expected behavior**
Import succeeds and Meilisearch does not crash.

**Screenshots**
![image](https://user-images.githubusercontent.com/2018660/121508296-1cf2b680-c9ee-11eb-9e0a-26ba8e2bde06.png)

**Additional context**
If you have any input on how I could provide better logs, please let me know.
